### ISSUE TITLE
Add support for adding single articles via https link

### ISSUE THREAD

**Issue by Gandalf-the-Blue** [Ashinch/ReadYou#128]
A way to add articles for reading later/starring via the https link for websites that do not offer RSS feeds/ for when you don't need the entire ongoing RSS feed.

**Comment by pc00per** [comment]
This isn't something that an RSS protocol is related to.

**Comment by Gandalf-the-Blue** [comment]
Agreed, but in terms of functionality and user requirements, it is essential.

**Comment by ayaen** [comment]
You wish to have a built-in bookmarks manager?
Cause that is what the above feature sounds like to me.
It maybe convenient of course to have all the things you wish to read in one place
but this function may be better served by a browser, no?

**Comment by Gandalf-the-Blue** [comment]
Reeder has the same functionality called iCloud Read Later - https://reederapp.com/
